wp-content/themes/citylimits/less/newsletters.less

```diff
@@ -196,6 +196,15 @@
 		margin-bottom: 10px;
 		padding: 4px 16px;
 	}
+	.uhohhoneypot {
+        opacity: 0;
+        position: absolute;
+        top: 0;
+        left: 0;
+        height: 0;
+        width: 0;
+        z-index: -1;
+    }
```

Comment:
Largo has a `.visuallyhidden` class that can be used here instead of adding our own.

Reply:
@benlk I figured Largo would have something like that, but wasn't sure if we should be concerned about using a class name with `hidden` in the name since I'm not sure if more advanced bots could detect that.

Reply:
I'd go with whatever Mailchimp provides in its default form markup, since that's probably backed by a lot of research.

Reply:
https://github.com/INN/umbrella-citylimits/pull/164/commits/e19e508c0dd9177bf9daa1eb966dcc884787f840 switches the fields to have the `.visuallyhidden` class